modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/TSLsAppris_conf.pm

```diff
@@ -63,122 +59,23 @@ sub default_options {
     return {
         # inherit other stuff from the base class
         %{ $self->SUPER::default_options() },
-        ensembl_release => 91, # Use it on the commandline: -ensembl_release XX
-        human_gencode_version => '25', # Use it on the command line: -human_gencode_version XX
-        mouse_gencode_version => 'M14', # Use it on the command line: -password mysql_rw_password
-        enscode_root_dir => '', #path to your perl modules
+        release => software_version(), # Use it on the commandline: -release XX
+        base_dir => '', #path to your perl modules
 #################
 #        Everything below should not need modification
 #################
         pipeline_name => 'tsl_appris_'.$self->o('ensembl_release'),
-
-        production_db_host => 'mysql-ens-sta-1',
-        production_db_port => 4519,
-        production_db_user => 'ensro',
-        production_db_password => undef,
-        production_db_driver => $self->o('hive_driver'),
-
-        data_db_host => $self->o('production_db_host'),
-        data_db_port => $self->o('production_db_port'),
-        data_db_user => $self->o('user'),
-        data_db_password => $self->o('password'),
-        data_db_driver => $self->o('production_db_driver'),
-
-        production_dir => catdir($self->o('enscode_root_dir'), 'ensembl-production'),
-
-        do_human => 1,
-        do_mouse => 1,
-        do_pig => 1,
-        do_chimpanzee => 1,
-        do_rat => 1,
-        do_zebrafish => 1,
-
-        human_alias => 'homo_sapiens',
-        human_cs_name => 'GRCh'.$self->o('human_cs_version'),
-        human_cs_version => '38',
-
-        mouse_alias => 'mus_musculus',
-        mouse_cs_name => 'GRCm'.$self->o('mouse_cs_version'),
-        mouse_cs_version => '38',
-
-        rat_alias => 'rattus_norvegicus',
-        rat_cs_name => 'Rnor_6.0',
-        rat_cs_version => 6,
-
-        zebrafish_alias => 'danio_rerio',
-        zebrafish_cs_name => 'GRCz11',
-        zebrafish_cs_version => 11,
-
-        chimpanzee_alias => 'pan_troglotydes',
-        chimpanzee_cs_name => 'Pan_tro_3.0',
-        chimpanzee_cs_version => 3,
-
-        pig_alias => 'sus_scrofa',
-        pig_cs_name => 'Sscrofa11.1',
-        pig_cs_version => 111,
-
-        tsl_ftp_base => 'http://hgwdev.cse.ucsc.edu/~markd/gencode/tsl-handoff',
+        muser     => undef,
+        mdbname   => undef,
+        mhost     => undef,
+        mport     => undef,
+        division => [],
+        history_file => undef,
+        production_dir => catdir($self->o('base_dir'), 'ensembl-production'),
+        tsl_ftp_base => 'http://hgwdev.gi.ucsc.edu/~markd/gencode/tsl-handoff/',
         appris_ftp_base => 'http://apprisws.bioinfo.cnio.es/forEnsembl',
-
         load_tsl_script => catfile($self->o('production_dir'), 'scripts', 'import_transcript_support_levels.pl'),
         load_appris_script => catfile($self->o('production_dir'), 'scripts', 'import_appris.pl'),
```

Comment:
These files are deleted now, I think, so presumably references to them can be deleted, to prevent future confusion?

Reply:
Well spotted @james-monkeyshines. I will clean this up.